galsim/gsobject.py

```diff
@@ -869,7 +840,7 @@ def _shift(self, offset):
         return new_obj
 
     # Make sure the image is defined with the right size and wcs for drawImage()
-    def _setup_image(self, image, nx, ny, bounds, add_to_image, dtype, odd=False, wmult=1.):
+    def _setup_image(self, image, nx, ny, bounds, add_to_image, dtype, odd=False, N=None, wmult=1.):
```

Comment:
After removing the wcs stuff from drawKImage, the `N` option here is never used.  I guess I should have removed it as part of that commit, but I didn't remember to.